<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="IN-L2504" ignored="0" name="GET Feeds - Duplicate Transactions" tests="16" failures="12" timestamp="14 Jun 2021 20:23:23 GMT" time="364.072" errors="0">
  <testcase name="testGetNotificationsForDuplicateTransaction" time="17.925" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions"/>
  <testcase name="testGetNotificationsForDuplicateTransaction" time="18.502" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions"/>
  <testcase name="testGetNotificationsForDuplicateTransaction" time="16.157" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions"/>
  <testcase name="testGetNotificationsForDuplicateTransaction" time="17.154" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
    <failure type="junit.framework.AssertionFailedError" message="Expected  2 insights triggered But found Empty {}">
      <![CDATA[junit.framework.AssertionFailedError: Expected  2 insights triggered But found Empty {}
	at junit.framework.Assert.fail(Assert.java:57)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForDuplicateTransaction -->
  <testcase name="testGetNotificationsForDuplicateTransaction" time="16.865" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
    <failure type="junit.framework.AssertionFailedError" message="Expected  2 insights triggered But found Empty {}">
      <![CDATA[junit.framework.AssertionFailedError: Expected  2 insights triggered But found Empty {}
	at junit.framework.Assert.fail(Assert.java:57)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForDuplicateTransaction -->
  <testcase name="testGetNotificationsForDuplicateTransaction" time="16.643" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.yodlee.insights.yodleeApi.utils.InsightsGenerics.getExpectedAttributesToBeValidatedV2(InsightsGenerics.java:1326)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:257)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForDuplicateTransaction -->
  <testcase name="testGetNotificationsForDuplicateTransaction" time="16.913" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
    <failure type="junit.framework.AssertionFailedError" message="Expected  2 insights triggered But found Empty {}">
      <![CDATA[junit.framework.AssertionFailedError: Expected  2 insights triggered But found Empty {}
	at junit.framework.Assert.fail(Assert.java:57)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForDuplicateTransaction -->
  <testcase name="testGetNotificationsForDuplicateTransaction" time="16.738" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
    <failure type="junit.framework.AssertionFailedError" message="Expected  2 insights triggered But found Empty {}">
      <![CDATA[junit.framework.AssertionFailedError: Expected  2 insights triggered But found Empty {}
	at junit.framework.Assert.fail(Assert.java:57)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForDuplicateTransaction -->
  <testcase name="testGetNotificationsForDuplicateTransaction" time="16.444" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
    <failure type="junit.framework.AssertionFailedError" message="Expected  2 insights triggered But found Empty {}">
      <![CDATA[junit.framework.AssertionFailedError: Expected  2 insights triggered But found Empty {}
	at junit.framework.Assert.fail(Assert.java:57)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForDuplicateTransaction -->
  <testcase name="testGetNotificationsForDuplicateTransaction" time="16.933" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
    <failure type="junit.framework.AssertionFailedError" message="Expected  2 insights triggered But found Empty {}">
      <![CDATA[junit.framework.AssertionFailedError: Expected  2 insights triggered But found Empty {}
	at junit.framework.Assert.fail(Assert.java:57)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForDuplicateTransaction -->
  <testcase name="testGetNotificationsForDuplicateTransaction" time="16.708" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
    <failure type="junit.framework.AssertionFailedError" message="Expected  2 insights triggered But found Empty {}">
      <![CDATA[junit.framework.AssertionFailedError: Expected  2 insights triggered But found Empty {}
	at junit.framework.Assert.fail(Assert.java:57)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForDuplicateTransaction -->
  <testcase name="testGetNotificationsForDuplicateTransaction" time="16.503" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
    <failure type="junit.framework.AssertionFailedError" message="Expected  2 insights triggered But found Empty {}">
      <![CDATA[junit.framework.AssertionFailedError: Expected  2 insights triggered But found Empty {}
	at junit.framework.Assert.fail(Assert.java:57)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForDuplicateTransaction -->
  <testcase name="testGetNotificationsForDuplicateTransaction" time="15.998" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.yodlee.insights.yodleeApi.utils.InsightsGenerics.getExpectedAttributesToBeValidatedV2(InsightsGenerics.java:1326)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:257)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForDuplicateTransaction -->
  <testcase name="testGetNotificationsForDuplicateTransaction" time="16.378" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.yodlee.insights.yodleeApi.utils.InsightsGenerics.getExpectedAttributesToBeValidatedV2(InsightsGenerics.java:1326)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:257)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForDuplicateTransaction -->
  <testcase name="testGetNotificationsForDuplicateTransaction" time="19.152" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
    <failure type="junit.framework.AssertionFailedError" message="Expected  2 insights triggered But found Empty {}">
      <![CDATA[junit.framework.AssertionFailedError: Expected  2 insights triggered But found Empty {}
	at junit.framework.Assert.fail(Assert.java:57)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForDuplicateTransaction -->
  <testcase name="testGetNotificationsForDuplicateTransaction" time="17.798" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions"/>
</testsuite> <!-- GET Feeds - Duplicate Transactions -->
